dev/benchmarks/complex_layout/pubspec.yaml

```diff
@@ -14,9 +14,9 @@ dependencies:
   #   //packages/flutter_tools/lib/src/commands/update_packages.dart
   # and run
   #   flutter update-packages --force-upgrade
-  flutter_gallery_assets: 1.0.2
+  flutter_gallery_assets: 1.0.1
```

Comment:
this got reverted? did you change this, or was it flutter update-packages?

Reply:
odd, i did that by accident at one point but i thought i'd fixed it... checking, one sec

Reply:
i think i forgot to rerun the tool after updating that particular pin, heh. will push new update.